app/services/cfe/submit_vehicle_service.rb

```diff
@@ -4,7 +4,8 @@ def call(cfe_assessment_id)
       return unless relevant_form?(:vehicle_details)
 
       details_model = ClientVehicleDetailsForm.from_session(@session_data)
-      vehicles = CfeParamBuilders::Vehicles.call(details_model, in_dispute: details_model.vehicle_in_dispute)
+      vehicles = CfeParamBuilders::Vehicles.call(details_model,
+                                                 in_dispute: details_model.vehicle_in_dispute && estimate.smod_applicable?)
```

Comment:
Only that, because it's only called once in this class, and is called explicitly rather than being inferred from a symbol representing the name, it would have involved more typing to add a `delegate` line than not. I don't think it really matters either way, but for me conciseness won out.